src/components/shared/Popover.js

```diff
@@ -24,6 +24,7 @@ class Popover extends Component {
     const left = targetLeft + targetWidth / 2 - width / 5;
     const top = targetBottom;
 
+    // eslint-disable-next-line react/no-did-mount-set-state
```

Comment:
Fix the following warning:
<img width="625" alt="screen shot 2017-03-07 at 10 34 54 am" src="https://cloud.githubusercontent.com/assets/4562118/23639680/fbd06d5c-0323-11e7-8724-4468d4385a1b.png">